tsconfig.json

```diff
@@ -10,6 +10,6 @@
     "outDir": "build",
     "resolveJsonModule": true
   },
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*.ts", "src/utils/typeGuards.ts", "src/utils/customErrors.ts"],
```

Comment:
The added cases are covered by the first statement. Might be an editor suggestion.